.github/workflows/scheduled.yml

```diff
@@ -704,6 +711,81 @@ jobs:
           path: |
             /tmp/aggregate_fuzzer_repro
             /tmp/server.log
+            
+
+  presto-bias-java-aggregation-fuzzer-run:
```

Comment:
Looking at ` presto-java-aggregation-fuzzer-run` it's not supposed to run on PRs is that the case here as well? Also do you want to run either or? Or just this one on PRs if a bias is detected?

Reply:
ATM 'presto-java-aggregation-fuzzer-run' is not supposed to run on PRs but we plan to change that as its been running green for a while. The bias one's can only run on PR

Reply:
ok, so we'll add the 'either-or-mechanic' when we activate it to run on PRs